demo/python/object_detection.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+
+import cv2
+from mmdeploy_python import Detector
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='show how to use sdk python api')
+    parser.add_argument(
+        'model_path', help='the directory path of mmdeploy model')
+    parser.add_argument('image_path', help='the path of an image')
+    parser.add_argument(
+        '--device-name', default='cpu', help='the name of device, cuda or cpu')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+
+    img = cv2.imread(args.image_path)
+    detector = Detector(args.model_path, args.device_name, 0)
+    bboxes, labels, masks = detector([img])[0]
+
+    indices = [i for i in range(len(bboxes))]
+    for index, bbox, label_id in zip(indices, bboxes, labels):
+        [left, top, right, bottom], score = bbox[0:4].astype(int), bbox[4]
+        if score < 0.3:
+            continue
+
+        cv2.rectangle(img, (left, top), (right, bottom), (0, 255, 0))
+
+        if masks:
```

Comment:
Mask is a list of empty ndarray, even when model is retinanet.

Reply:
`mask` is only for instance seg.

Reply:
object_detection failed because retinanet will also enter this branch.